### ISSUE TITLE
Mobile Menus

### ISSUE THREAD

**Issue by jcsparks** [CuBoulder/ucb2021_base#120]
Evaluate https://mmenujs.com/ for implementation of mobile menus in the D9 theme. 

Need to ensure a11y issues are met, at first glance this seems to be a11y aware.  

As much as possible, also determine if this will cause any migration issues.

**Comment by jnicholCU** [comment]
Accessibility-wise it seems that he has added a lot more functionality that people were requesting in the newer 9.0 versions.
The main problem I found was that the hamburger button and X button didn't have any aria-hidden or other labeling so keyboard users would lose their focus. Since the fix it looks like the wave tool isn't picking up any more issues.

The only other thing to keep in mind is his demo site doesn't have the accessibility options enabled and he is looking to update the site at some point soon

**Comment by jnicholCU** [comment]
After a lot of testing (too much testing?) I don't think this mobile menu option is going to work. It works decently when not logged in but the admin toolbar breaks in weird ways when I enable the keyboard menuing/accessibility options. Also get some of the errors I saw just in the demo site with burger menu/submenu not working until site is resized or reloaded. I posted another ticket so we'll see if I hear back. Looked through some other tickets to see if there were any Drupal users and all of them were Drupal 7 so not too helpful. So yeah, overall was too much finagling to get it to work, and not even fully work. Will update the issue in github.
Would be worth looking into other options for mobile menuing though.
